xlators/debug/io-stats/src/io-stats.c

```diff
@@ -1398,6 +1404,19 @@ io_stats_dump_global_to_logfp(xlator_t *this, struct ios_global_stats *stats,
                     gf_upcall_list[i], fop_hits, "0", "0", "0");
     }
 
+    /* To measure FOP_HITS_PER_SEC consider time difference
+       between last incremental profile stat started and last
+       fop hit.
+    */
+    duration = (conf->last_fop_hit - stats->started_at);
```

Comment:
`duration` is an unsigned integer, so if `last_fop_hit` is 0 or less than `started_at` (which I think it can happen if no fop has been received), the shown value will be completely wrong.

Reply:
done